timemachine/fe/free_energy.py

```diff
@@ -340,6 +354,8 @@ def sample(initial_state: InitialState, md_params: MDParams, max_buffer_frames:
         store_x_interval=0,
     )
 
+    rng = np.random.default_rng(md_params.n_frames * md_params.steps_per_frame)
```

Comment:
Why is this not just init'd to the seed?

Reply:
Seed is not available through InitialStates or the MDParams. Should we add the seed on the InitialState?

Reply:
Maybe add to MDParams ?

Reply:
Added in https://github.com/proteneer/timemachine/pull/1026/commits/cd1d8faf7a17f3322d89f0618577640718ce2c1a